consumer/dragonboard410c/guides/aosp.md

```diff
@@ -15,69 +15,31 @@ The following instructions can be used to attempt a build based on the AOSP mast
 
 In this page, we assume that `<ANDROID_TOP>` is the top level folder for your Android environment. e.g.:
 ```shell
-mkdir <ANDROID_TOP>
-cd <ANDROID_TOP>
+$ mkdir <ANDROID_TOP>
+$ cd <ANDROID_TOP>
 ```
 Then run:
 ```shell
-repo init -u https://android.googlesource.com/platform/manifest -b master
-cd .repo
-git clone https://github.com/robherring/android_manifest.git -b master local_manifests
-cd ..
-repo sync -j24
+$ repo init -u https://android.googlesource.com/platform/manifest -b master
+$ git clone https://android-git.linaro.org/git/platform/manifest.git .repo/local_manifests -b dragonboard
+$ repo sync -jX -c # X being the number of logical cores in you build system
```

Comment:
i like to recommend -j`nproc` in such situations.

Reply:
ok i'll add -j$(nproc)